public/data/Bonganibendi.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "Bongani Bendile",
+  "type": "personal",
+  "bio": "I'm a finance and investment management student enthusiastic about the intersection of finance and software",
+  "avatar": "https://avatars.githubusercontent.com/u/58882596?s=400&u=85fa39e4c5bf28dcce96a22eae70f1a8e45565f2&v=4",
```

It is best to use it this way, to have the link shorter and work better.
```suggestion
  "avatar": "https://github.com/Bonganibendi.png",
```